src/data/faq.json

```
@@ -145,6 +145,23 @@
                             "Under the tab 'Certificates' the app can only scan '<b>EU Digital</b> COVID  Certificates'.",
                             "See also the following entry on <a href='#vac_cert_invalid' target='_self' rel='noopener'>invalid vaccination certificates</a>."
                         ]
+                    },
+                    {
+                        "title": "How long is my certificate valid​",
+                        "anchor": "eu_dcc_validity",
+                        "active": true,
+                        "textblock": [
+                            "The following text refers to EU digital COVID certificates issued in Germany, i.e. vaccination, test and recovery certificates. The figures given are the status as of July 2021.",
+                            "Basically, we distinguish<ul><li>the validity of the digital certificate (the <b>technical validity</b>) and</li><li>the validity of the vaccination protection (the <b>effective vaccination validity</b>), i.&nbsp;e. the 'duration of protection'</li></ul>",

```

```suggestion
                            "Basically, we distinguish between<ul><li>the validity of the digital certificate (the <b>technical validity</b>) and</li><li>the validity of the vaccination protection (the <b>effective vaccination validity</b>), i.e. the 'duration of protection'</li></ul>",
```
https://dictionary.cambridge.org/dictionary/english/distinguish (between)
and
remove &nbsp; from English abbreviation.